daemon_windows.go

```diff
@@ -27,37 +29,66 @@ func newDaemon(name, description string, dependencies []string) (Daemon, error)
 func (windows *windowsRecord) Install(args ...string) (string, error) {
 	installAction := "Install " + windows.description + ":"
 
-	return installAction + failed, ErrWindowsUnsupported
+	execp, err := execPath()
+
+	if err != nil {
+		return installAction + failed, err
+	}
+
+	cmd := exec.Command("nssm.exe install Description=\"" + windows.description + "\" \"" + windows.name + "\" " + execp)
+	err = cmd.Run()
+	if err != nil {
+		return installAction + failed, err
+	}
+
+	return installAction + " completed.", nil
 }
```

Comment:
This isn't the best way. But I think it will work. I am not sure about `Descriptilon` parameter. I have to ry it after switching to windows.